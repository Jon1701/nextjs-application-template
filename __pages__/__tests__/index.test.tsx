import React from 'react';
import { describe, test, expect } from '@jest/globals';
import { render } from '@testing-library/react';

import IndexPage from '@pages/index';

describe('IndexPage', () => {
  test('should render correctly', () => {
    const node = render(<IndexPage />);

    expect(node).toMatchSnapshot();
  });
});
