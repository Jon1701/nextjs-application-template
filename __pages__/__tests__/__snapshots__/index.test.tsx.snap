// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage should render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="sc-bcXHqe elINVw"
      >
        <h1
          class="sc-gswNZR"
        >
          Next.js Application Template
        </h1>
        <p>
          This is a template repository to quickly start developing Next.js and TypeScript applications.
        </p>
      </main>
    </div>
  </body>,
  "container": <div>
    <main
      class="sc-bcXHqe elINVw"
    >
      <h1
        class="sc-gswNZR"
      >
        Next.js Application Template
      </h1>
      <p>
        This is a template repository to quickly start developing Next.js and TypeScript applications.
      </p>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
